aws/provider.go

```diff
@@ -209,6 +209,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_ebs_snapshot":                              dataSourceAwsEbsSnapshot(),
 			"aws_ebs_snapshot_ids":                          dataSourceAwsEbsSnapshotIds(),
 			"aws_ebs_volume":                                dataSourceAwsEbsVolume(),
+			"aws_ebs_volume_ids":                            dataSourceAwsEbsVolumeIDs(),
```

Comment:
Since these plural data sources tend to accumulate additional information over time (e.g. ARNs), we have preferred to name the new data source types without any initial attribute in the type. 👍 

```suggestion
			"aws_ebs_volumes":                               dataSourceAwsEbsVolumes(),
```

If you could update all the file/function names as well, that would be great. Thanks!